controllers/feeds/getFeedChat.js

```diff
@@ -113,7 +114,7 @@ const pushToa = (a, b, newGroup, actor, data) => {
   }
 };
 
-const finalize = (req, id, myReaction, newGroup, activity_id, constantActor) => {
+const finalize = (req, id, myReaction, newGroup, activity_id, constantActor, karmaScores = []) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `karmaScores` parameter to the `finalize` function is a good change. This allows the function to update the karma scores for each user, enhancing the functionality of the application.
```diff
+ const finalize = (req, id, myReaction, newGroup, activity_id, constantActor, karmaScores = []) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->